.travis.xacro_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
So please use `/opt/ros/$ROS_DISTRO/lib/xacro/xacro` instead of `rosrun xacro xacro`